source/server/admin/prometheus_stats.cc

```diff
@@ -146,6 +148,14 @@ std::string generateNumericOutput(const StatType& metric,
   return fmt::format("{0}{{{1}}} {2}\n", prefixed_tag_extracted_name, tags, metric.value());
 }
 
+std::string generateTextReadoutOutput(const Stats::TextReadout& text_readout,
+                                      const std::string& prefixed_tag_extracted_name) {
+  auto tags = std::vector(text_readout.tags());
```

Comment:
You're right, it was unnecessary. I wanted to copy tags to prevent modifying them and I forgot that tags are returned by value, not by reference. I removed this constructor invocation.